openfeature/event_executor.go

```diff
@@ -110,8 +133,8 @@ func (e *eventExecutor) removeApiHandler(t EventType, c EventCallback) {
 	e.apiRegistry[t] = entrySlice
 }
 
-// registerClientHandler registers a client level handler
-func (e *eventExecutor) registerClientHandler(clientName string, t EventType, c EventCallback) {
+// RegisterClientHandler registers a client level handler
+func (e *EventExecutor) RegisterClientHandler(clientName string, t EventType, c EventCallback) {
```

Comment:
maybe name this `AddClientHandler` to reflect the relationship to `AddHandler` (i.e. the same except scoped), or perhaps rename `AddHandler` above as `RegisterHandler`?

Reply:
will rename to `AddClientHandler`. Regarding `AddHandler`, this is already public so I will avoid a breaking change and keep it there.